srml/balances/src/lib.rs

```diff
@@ -217,10 +225,13 @@ impl<T: Trait> Module<T> {
 		}
 	}
 
-	/// Set the free balance of an account to some new value.
+	/// Set the reserved balance of an account to some new value. Will enforce `ExistentialDeposit`
+	/// law, annulling the account as needed.
 	///
-	/// Will enforce ExistentialDeposit law, anulling the account as needed.
-	/// In that case it will return `AccountKilled`.
+	/// Doesn't do any preparatory work for creating a new account, so should only be used when it
+	/// is known that the account already exists.
+	///
+	/// Returns either `Updated` or `AccountKilled`.
```

Comment:
You describing code. If the user wants to know the variants of `UpdateBalanceOutcome`, he will click on it. 
I think the return value name is self descriptional enough to drop this comment.